src/pybricks/hubs.pyi

```diff
@@ -35,6 +35,15 @@ class TechnicHub:
     imu: IMU
 
 
+class EssentialHub:
+    def __init__(self, top_side: Axis = Axis.Z, front_side: Axis = Axis.X): ...
+    battery: Battery
+    buttons: Keypad
```

Comment:
I see!

But this is even more confusing to me in that case. It is a singular word `button` but the `button()` function returns a Tuple with a single element only, rather than a boolean indicating what's the button's state.

Reply:
I agree, but it has already been implemented this way on the other hubs that only have one button for a long time. And `if hub.button.pressed():` still works as expected because of the Python concept of truthy and falsy (non-empty tuple is truthy and empty tuple is falsy).